content/warnings/dont-call-proptypes.md

```diff
@@ -92,9 +92,9 @@ export default function deprecated(propType, explanation) {
       }
     }
 
-    return propType(props, propName, componentName, ...rest); // and here
+    return propType(props, propName, componentName, ...rest); // và ở đây
   };
 }
 ```
 
-This will silence the warning.
+Điều này sẽ tắt tiếng cảnh báo.
```

Comment:
Warning này là log warning ở console nha em, em thay đổi lại giúp a chỗ này nhé. Khúc này nếu đổi lại sẽ phù hợp hơn nè

Reply:
Điều này sẽ tắt log warning ở console

Như này được không anh?

Reply:
"Điều này sẽ giúp xoá warning ở console"

Như vầy được ko em nè?

Reply:
Điều này sẽ ngưng in ra cảnh báo ở console

Hay như này ạ!

Reply:
Làm mất cảnh báo thôi em, "ngưng" hoặc "tắt" nghĩa là tắt luôn rồi, Anh sợ sẽ dễ hiểu lầm là tắt luôn log

Reply:
Dạ em vừa update ạ. Cảm ơn anh nhiều hehe